playground/backend/pkg/api/pathchecker/pathchecker.go

```diff
@@ -0,0 +1,49 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pathchecker utils for checking the valid file path
+package pathchecker
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// IsExist checks if file exists
+func IsExist(filePath string) (bool, error) {
+	if _, err := os.Stat(filePath); err == nil {
+		return true, nil
+	} else if os.IsNotExist(err) {
+		return false, nil
+	} else {
+		return false, err
```

Comment:
What does the error mean in this case? If not exist? We return false in this case.

Reply:
Nope, if not exists it is the second If
This one if something went wrong with the check itself.

Reply:
what does mean check itself? :)